server/game/player.js

```diff
@@ -287,17 +283,26 @@ class Player {
     }
 
     revealPlot() {
-        this.selectedPlot.facedown = false;
-
         this.menuTitle = '';
         this.buttons = [];
 
-        this.plotDiscard.push(this.selectedPlot.card);
+        this.selectedPlot.facedown = false;
+        if (this.activePlot !== null && this.activePlot !== undefined) {
```

Comment:
the null check is probably unnecessary(I try to avoid setting stuff to null wherever possible), if(this.activePlot) or !_.isUndefined(this.activePlot) should suffice